recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,19 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,16 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if not "PYSDL2_DLL_PATH" in os.environ:
```

Comment:
Thanks for looking again! A long time ago, I remember that the `... not in ...` syntax caused a syntax error, so I taught myself never to use it and stick with this one. But that may have been around Python 2.6, and I definitely agree a `not in` construct is cleaner, so I'll change it in a final commit (hopefully ;) ).